upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -1183,6 +1094,8 @@ if [[ ${state_recorded} == "0" ]]; then
   {
     export PDSH_SSH_ARGS_APPEND="-o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null"
     rpm --force -Uvh "$(find "${CSM_ARTI_DIR}"/rpm/cray/csm/ -name \*csm-testing\*.rpm | sort -V | tail -1)"
+    rpm --force -Uvh "$(find "${CSM_ARTI_DIR}"/rpm/cray/csm/ -name \*platform-utils\*.rpm | sort -V | tail -1)"
```

Comment:
Should we also be updating goss-servers to make sure it matches csm-testing?

Reply:
Example from surtur:
```
ncn-m001:~ # rpm -qa | grep csm-testing
csm-testing-1.16.36-1.noarch
ncn-m001:~ # rpm -qa | grep goss-servers
goss-servers-1.15.46-1.noarch
```

Reply:
I was just going to suggest the same based on https://github.com/Cray-HPE/node-images/blob/main/vendor/github.com/Cray-HPE/metal-provision/roles/node_images_ncn_common/files/scripts/metal/metal-lib.sh#L499

Reply:
Thanks. Fixed.